### ISSUE TITLE
Créer une page permettant de démarrer/éteindre une VM

### ISSUE THREAD

**Issue by ChamMach** [ChamMach/VirtualWeb#20]

**Comment by baptistebisson** [comment]
https://github.com/ChamMach/VirtualWeb/commit/e37bd04133e221674d890956b8689a281a0bef28 : 
- L'action démarrer et éteindre la VM est relié à une requête ajax. Le retour est comme toujours normalisé : 
`{'message': 'mon message', 'erreur' : true/false}`
J'ai aussi pris en compte les messages de retour de #25

**Comment by baptistebisson** [comment]
https://github.com/ChamMach/VirtualWeb/commit/2d10bb8383ac733547b7669e492676eefa9dfa7a
J'ai pu corriger mon script d'import en utilisant ton exe à jour. Le comportement est le suivant : 
- Je récupère toutes les VM en base. Si une VM n'est pas présente dans le retour JSON --> suppression (normalement ce n'est pas censé arriver, car la VM ne doit être supprimée que par une requête ajax de l'utilisateur, mais je préfère prévoir le coup histoire de ne pas avoir de VM en base qui n'existe pas en vrai)
- Si l'utilisateur existe bien en base on fait un test pour voir si la VM existe. Si ce n'est pas le cas, on la crée, sinon on la met à jour.

>La VM ne peut pas se mettre à jour si le nom de la VM n'est pas le même que celui déjà présent en base. Il risque donc d'y avoir un doublon en base, car la comparaison ce fait sur l'id utilisateur et le nom.


- L'utilisateur peut lancer et éteindre une VM depuis l'espace dédié. Il reçoit un message de succès ou d'erreur selon le statut reçu par le JSON. Il faudra que je rajoute un loader pour faire patienter la personne.

D'ailleurs mon script attend 7 secondes avant la réponse avant de fermer le socket. Ce qui doit être largement suffisant.

**Comment by ChamMach** [comment]
Super ! Je testerais tout cela ce week-end, bon boulot @baptistebisson !

**Comment by ChamMach** [comment]
Plop, j'ai testé et le démarrage et l'arrêt d'une VM fonctionne très bien, bon boulot ! En revanche, une fois la VM éteinte le statut de celle-ci reste en "En service" : 
![image](https://user-images.githubusercontent.com/23166349/34905402-2a80c58c-f858-11e7-99f3-5182d581febf.png)

Une idée @baptistebisson ?

**Comment by baptistebisson** [comment]
Il faut actualiser la page 🦅

**Comment by ChamMach** [comment]
Déjà fait plusieurs fois 💯
